PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,70 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the the technology which allows users to get access to smart points of interaction so that they can obtain extra information (infotainment, …), provide suggestions (suggestions mailbox, …) or generate new content (co-creation, …). The data model contains resources to configure the interaction service such as the broadcasted URL (typically shortened), the interval between broadcasts (frequency), the availability of the service, transmission power depending on the area to be covered, etc.
```

Comment:
I would use "etc." instead of "..." if any. It felt weird to me, but just an opinion.